Dockerfile

```diff
@@ -1,4 +1,12 @@
-FROM rust:1.59.0-bullseye
+FROM rust:1.59.0-bullseye as build
```

Comment:
There is a `docker-compose.yml` file which uses this Dockerfile where no multi stage were declared. The docker-compose file was used for dev purposes (i.e. to create a docker environment on the developer machine in order to develop this tool).

I don't know how you'd like the development process to be, but I'd keep it dockerized IMHO, with a bind mount of the project dir. This could be done via updating the `Dockerfile` and `docker-compose.yml` file by declaring a `base` stage for instance.

e.g. :

```
# Dockerfile
FROM rust:1.59.0-bullseye as base

RUN apt-get update \
    && apt-get install -y \
        git \
    && rm -rf /var/lib/apt/lists/*

WORKDIR /app

USER 1000

FROM base as build

COPY . .

RUN cargo build --release

FROM base as release

COPY --from=build /app/target/release/ssc /usr/bin/ssc

CMD ["ssc"]
```

```
# docker-compose.yml
---
version: '3.7'

services:
  app:
    build:
      context: .
      target: base
    environment:
      - CARGO_HOME=/app/.cargo
      - "USER=KNP Labs"
    volumes:
      - "./:/app"
```

Did you have an other idea in mind for the development process ?